TAS_TBDAdapter/TASTBDAdapter.cs

```diff
@@ -37,10 +40,19 @@ public partial class TasTBDAdapter : BHoMAdapter
         /**** Constructors                              ****/
         /***************************************************/
 
-        public TasTBDAdapter(string tBDFilePath = "")
+        [Description("Produces an TAS Adapter to allow interopability with TAS tBD files and the BHoM")]
+        [Input("tBDFilePath", "Path to tBD file")]
+        [Input("tasSettings", "Input additional settings the adapter should use")]
+        public TasTBDAdapter(string tBDFilePath = "", TASSettings tasSettings = null)
         {
+            if(tasSettings == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Please set some TAS Settings on the TAS Adapter before pushing");
```

Comment:
These settings are also used on pull, I would remove the last 2 words of the error.